zpeclient.go

```diff
@@ -328,8 +332,9 @@ func (c *Cache) authorize(principal string, check AthenzAccessCheck) (bool, erro
 	for role, members := range domainData.roleToPrincipals {
 		for _, member := range members {
 			if member.memberRegex.MatchString(principal) {
-				if member.expiration.IsZero() || member.expiration.After(time.Now()) {
+				if (member.expiration.IsZero() || member.expiration.After(time.Now())) && !member.systemDisabled {
 					roles = append(roles, role)
+					break
```

Comment:
We can't do a break here because the member can be part of multiple different roles and only one of them might have the permissions to give access. With this change, users will actually be denied unless that first role has the policy to allow them.

Reply:
My bad, you're right. Removed the `break` - https://github.com/yahoo/k8s-athenz-webhook/pull/24/commits/861591ef716779739bee91912b67c44ceb37ab39.